### DISCUSSION TITLE
Add ->live() and reactive methods

### DISCUSSION THREAD

**Discussion by SmartVolt** [malzariey/filament-daterangepicker-filter#37] [Ideas]
Hi, and thanks for this plugin!
I'd like tu use daterangepicker in form filelds, but i want to update states and lauch livewire dispatch: 
i'm building a custom page in filament where i select a date range and widgets updates automatically.
For now i'm using:
`DatePicker::make('from')
->live()
->afterStateUpdated(fn (?string $state) => $this->dispatch('updateFromDate', from: $state)),`
But i'd like to use DateRangeFilter for this.

Best regards
Luigi

**Comment by malzariey** [comment]
I believe DateRangePicker will work live() as default. As I am changing the value of the state intently after you choose a range in the picker.
